Cloudinary/Cloudinary.cs

```diff
@@ -382,8 +382,9 @@ public RenameResult Rename(string fromPublicId, string toPublicId, bool overwrit
         /// <returns></returns>
         public RenameResult Rename(RenameParams parameters)
         {
-            string uri = m_api.ApiUrlImgUpV.Action("rename").BuildUrl();
-
+            string uri = m_api.ApiUrlImgUpV.ResourceType(
+				Api.GetCloudinaryParam<ResourceType>(parameters.ResourceType)).
```

Comment:
Is "ResourceType" is a required parameter/property? 
if so, how did it work before? (was there a default resource type...I guess ...image)
if not, what happens if the value is null (meaning the user didn't set it)?